src/main/webapp/WEB-INF/themes/mobile/resource/css/interface.css

```diff
@@ -240,6 +242,10 @@
   margin-top: 0 !important;
 }
 
+.amendment-correction {
```

Comment:
All amendments currently have a red background on mobile, in contrast with desktop.  This brings mobile to parity with desktop where corrections are off-white, retractions and EOCs are red, and others are a journal-branded shade

Reply:
I went a little rogue and changed corrections to off-white for parity with desktop. They were previously red like retractions. Just checking if I should have kept them red in mobile? @samV-plos